README.md

```diff
@@ -2,33 +2,40 @@
 
 ## Installation
 
-We provide binaries for the popular platforms at our [Releases](https://github.com/hazelcast/hazelcast-commandline-client/releases) page.
-In order to install CLC:
-
-* Download the release package for your platform,
-* Extract it,
-* Optionally move the `clc` binary to somewhere in your *PATH*, so it can be run in any terminal without additional settings.
-
 Currently we provide precompiled binaries of CLC for the following platforms and architectures:
 
 * Linux/amd64
 * Windows/amd64
 * MacOS/amd64
 * MacOS/arm64
 
```

Comment:
The official name for the OS should be "macOS", as far as I know.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/377/commits/d919d6185f9805aa30eb423d83a3272c360b2f8b